### ISSUE TITLE
Variable batch size when exporting PyTorch to ONNX

### ISSUE THREAD

**Issue by HaziqRazali** [onnx/onnx#1195]
I am trying to import a model that was trained in PyTorch into Tensorflow. My plan is to export the model from PyTorch to ONNX then import it to Tensorflow. I am following this tutorial https://github.com/onnx/tutorials/blob/master/tutorials/PytorchOnnxExport.ipynb but noted that the batch size is specified as 1. Is it possible to have the batch size set to `None`  like in Tensorflow ?

**Comment by ebarsoum** [comment]
Yes, for ONNX a string value for the axis mean variable. Can you open an issue on PyTorch?
